src/netex_idf/offers.rs

```diff
@@ -0,0 +1,79 @@
+use crate::{model::Collections, Result};
+use failure::format_err;
+use log::{info, warn};
+use minidom::Element;
+use std::{fs::File, io::Read, path::Path};
+use walkdir::WalkDir;
+
+pub fn read_offer_folder(offer_folder: &Path, _collections: &mut Collections) -> Result<()> {
+    let calendrier_path = offer_folder.join("calendriers.xml");
+    if calendrier_path.exists() {
+        let mut calendriers_file = File::open(&calendrier_path)?;
+        let mut calendriers_file_content = String::new();
+        calendriers_file.read_to_string(&mut calendriers_file_content)?;
+        let calendriers: Element = calendriers_file_content
+            .parse()
+            .map_err(|_| format_err!("Failed to open {:?}", calendrier_path))?;
+        info!("Reading {:?}", calendrier_path);
+        parse_calendriers(&calendriers)?;
+    } else {
+        warn!(
+            "Offer {:?} ignored because it does not contain the 'calendrier.txt' file.",
```

Comment:
calendriers.xml and not .calendrier.txt.
You can use a variable to prevent that.

Reply:
Done in 8ebcb63c268de7fdcaaf9462bce62c788235ba0a.